pom.xml

```diff
@@ -105,44 +106,45 @@
     <dependency>
       <groupId>com.google.truth</groupId>
       <artifactId>truth</artifactId>
-      <version>0.42</version>
+      <!--Version 1.0 is expected to release on July 8-->
+      <version>0.46</version>
```

Comment:
Now you've added a property for `truth.version` you can use it here and below by `${truth.version}`